src/test/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ObjectionServiceTest.java

```diff
@@ -69,6 +74,8 @@
     private static final String ATTACHMENT_ID = "12348765";
     private static final String REASON = "REASON";
     private static final String ACCESS_URL = "/dummyUrl";
+    private static final Long ACTION_CODE_OK = 3000L;
```

Comment:
Are these made up action/ineligibility codes for test purposes?

Reply:
yes, it's just to test that the same value is being populated in the objection that is saved to mongo.